### DISCUSSION TITLE
What is command `foo` and `bar`

### DISCUSSION THREAD

**Discussion by adminerest** [chr233/ASFEnhance#91] [提问求助 / Support] [answered]
<img width="981" alt="截屏2023-03-19 19 23 24" src="https://user-images.githubusercontent.com/43260539/226172061-6071b7a6-d5e5-4adf-af9c-a223d64c3aa8.png">
Hello, when i read the `README.md` today, I saw there is a config for disabling commands. But I couldn't find any wiki which tell me what is the function of the default banned commands `foo` or `bar`. Could you tell me what these two commands do? Thanks.   
我今天在看底下readme时候看见有一个屏蔽命令的设置，但是里面默认值`foo`和`bar`这俩命令我没找到在哪里有说干啥用的。。。能否告知一下这俩命令的作用或者有其他地方有解释这两个命令的话贴个链接也行，感谢

**Comment by chr233** [comment, marked as answer]
没有这两个命令，就是用来举例子的
